src/config.js

```diff
@@ -261,5 +261,15 @@ export const NO_WARNING_BANNER_HOSTNAMES = ['test.map.geo.admin.ch', 'map.geo.ad
  *
  * @type {String[]}
  */
-
 export const WARNING_RIBBON_HOSTNAMES = ['test.map.geo.admin.ch']
+
+/**
+ * To avoid breaking legacy KML drawing during an MVP (test phases) we disable the drawing menu for
+ * those ones on test.map.geo.admin.ch
+ *
+ * @type {String[]}
+ */
+export const DISABLE_DRAWING_MENU_FOR_LEGACY_ON_HOSTNAMES = [
+    'test.map.geo.admin.ch',
+    'sys-map.dev.bgdi.ch',
```

Comment:
is it wanted that INT and PROD are excluded here?

Reply:
Actually I wanted to have the protection only on `test.map.geo.admin.ch`. On prod it is on purpose to not have the protection as as soon we go to PROD we don't want this protection anymore ? I added DEV in order to test the protection.